packages/ckeditor5-source-editing/docs/features/source-editing.md

```diff
@@ -11,7 +11,7 @@ The {@link module:source-editing/sourceediting~SourceEditing} feature provides t
 The changes made to the document source will be applied to the editor's {@link framework/guides/architecture/editing-engine data model} only, if the editor understands (via loaded plugins) the given syntax. You will lose all changes that the editor features cannot understand. For example, if the editor does not have a {@link features/horizontal-line horizontal line} plugin loaded, the `<hr>` tag added in the document source will be removed upon exit from the source editing mode.
 
 <info-box>
-	Currently, the source editing mode is supported only for the {@link examples/builds/classic-editor classic editor}.
+	Currently, the source editing mode is supported only for the {@link examples/builds/classic-editor classic editor}. The source editing feature may not work with [CKEditor 5 collaboration features](https://ckeditor.com/docs/ckeditor5/latest/features/collaboration/collaboration.html).
```

Comment:
`may not work` -> `does not work`? `supported for` -> `supported by`?

Reply:
I'd say "supported in the classic editor".  
  
As for "may" it depends - if it never works, then yes, "does not work".

Reply:
I would leave the first suggestion ("_may not work_") as it is now, because in some cases `SourceEditing` will work with CF plugins.

I've changed the second one ("_supported only for_" -> "_supported only by_") as you proposed.

Reply:
Now I noticed a comment from @godai78, sorry 🙂 I changed this sentence as proposed: "_Currently, the source editing mode is supported in the classic editor._"